pkg/webui/console/components/user-data-form/index.js

```diff
@@ -49,6 +58,10 @@ const validationSchema = Yup.object().shape({
     .oneOf(approvalStates)
     .required(sharedMessages.validateRequired),
   description: Yup.string().max(2000, Yup.passValues(sharedMessages.validateTooLong)),
+  password: Yup.string().required(sharedMessages.validateRequired),
```

Comment:
```suggestion
  password: Yup.string().min(8, Yup.passValues(sharedMessages.validateTooShort)).required(sharedMessages.validateRequired),
```

To make it match the validation in `oauth/views/create-account.js`

Reply:
I think it's better not to do this for now until #2913  - this way, you get an error back from the NS explaining the failure based on the config. Until #2913 we don't have a way to make the regex match the config

Reply:
Whatever we do there, it needs to be consistent in all forms that have password fields.

Reply:
any validation we put here could get out of sync with config. until #2913 is done, i think we have to remove validation from any config controlled password fields. @htdvisser ?